internal/infrastructure/router/public.go

```diff
@@ -69,7 +70,27 @@ func newRbacSkipper(service string) echo_middleware.Skipper {
 		panic("service is an empty string")
 	}
 	return func(c echo.Context) bool {
-		return c.Path() == "/api/"+service+"/v1/openapi.json"
+		ctx := c.Request().Context()
+		// The access to the openapi specification is public
+		if c.Path() == "/api/"+service+"/v1/openapi.json" {
+			slog.DebugContext(ctx, "Skipping "+c.Path())
+			return true
+		}
+		cc, ok := c.(middleware.DomainContextInterface)
+		if !ok {
+			slog.WarnContext(ctx, "'c' is not a DomainContextInterface")
+			return false
```

Comment:
thanks for the suggestion! it can happen in a bug! indeed this PR fix a situation detected when adding the smoke test for the System endpoints. I suggest to keep the return as `false`, as a protection when in the future the API changes or extends.